fonts.css

```diff
@@ -0,0 +1,27 @@
+@font-face {
+  font-family: 'PT Serif';
+    src:
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/MyBali/pullRequest?prid=5764748)